dsc/tests/dsc_export.tests.ps1

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'resource export tests' {
+    
+    It 'Export can be called on individual resource' {
+
+        $out = dsc resource export -r Microsoft/Process
+        $LASTEXITCODE | Should -Be 0
+        $config_with_process_list = $out | ConvertFrom-Json
+        $config_with_process_list.resources.count | Should -BeGreaterThan 1
+    }
+
+    It 'get --all can be called on individual resource' {
+
+        $out = dsc resource get --all -r Microsoft/Process
+        $LASTEXITCODE | Should -Be 0
+        $process_list = $out | ConvertFrom-Json
+        $process_list.resources.count | Should -BeGreaterThan 1
+    }
+
+    It 'Export can be called on a configuration' {
+
+        $yaml = @'
+            $schema: https://schemas.microsoft.com/dsc/2023/03/configuration.schema.json
+            resources:
+            - name: Processes
+              type: Microsoft/Process
+              properties:
+                pid: 0
+'@
+        $out = $yaml | dsc config export
+        $LASTEXITCODE | Should -Be 0
+        $config_with_process_list = $out | ConvertFrom-Json
+        $config_with_process_list.resources.count | Should -BeGreaterThan 1
```

Comment:
some validation that the output is a config doc

Reply:
Updated.